content/tutorials/manage-the-rippled-server/configuration/run-rippled-in-reporting-mode.md

```diff
@@ -0,0 +1,375 @@
+---
+html: run-rippled-in-reporting-mode.html
+parent: configure-rippled.html
```

Comment:
This one is a weird case because it's part installation, part configuration. My thought is that since it doesn't build on the standard install, this tutorial should be viewed as more of a non-standard installation option, so I would put it in the "Install rippled" category rather than the "Configure rippled" category.